release/release-notes/data-prepper.release-notes-1.2.0.md

```diff
@@ -0,0 +1,63 @@
+* New getting started, developer, getting started trace analytics and pipeline setup guide ([#346](https://github.com/opensearch-project/data-prepper/pull/346))
+* Added new log ingestion guide showcasing new HTTP and Grok Processor Plugins ([#573](https://github.com/opensearch-project/data-prepper/pull/573))
+* New guide for migrating to OpenSearch Data Prepper from Open Distro Data Prepper ([#470](https://github.com/opensearch-project/data-prepper/pull/470))
+* Added Project Resource to documentation ([#482](https://github.com/opensearch-project/data-prepper/pull/482))
+* Added Coding Guidance to the Developer Guid ([#560](https://github.com/opensearch-project/data-prepper/pull/560))
+* Added instructions to build and run the Docker image locally ([#564](https://github.com/opensearch-project/data-prepper/pull/564))
+* Updated copyright headers for root project, api and core ([#569](https://github.com/opensearch-project/data-prepper/pull/569))
+* Update documentation to use OpenSearch Dashboards ([#658](https://github.com/opensearch-project/data-prepper/pull/658))
+* Improving OpenSearch sink documentation ([#553](https://github.com/opensearch-project/data-prepper/pull/553)), ([#562](https://github.com/opensearch-project/data-prepper/pull/562)), & ([#563](https://github.com/opensearch-project/data-prepper/pull/563))
+
+### Maintenance
+* Updated version to 1.2 ([#416](https://github.com/opensearch-project/data-prepper/pull/416))
+* The OpenSearch REST client and the build plugins used by the OpenSearch plugin are now at 1.1.0 ([#384](https://github.com/opensearch-project/data-prepper/pull/384))
+* Use OpenSearch instead of Elasticsearch in builds ([#438](https://github.com/opensearch-project/data-prepper/pull/438))
+* Use Netty 4.1.68 which fixes CVE-2021-37136 and CVE-2021-37137 ([#661](https://github.com/opensearch-project/data-prepper/pull/661))
+* Uses Log4j 2.16.0 which fixes CVE-2021-44228 ([#742](https://github.com/opensearch-project/data-prepper/pull/742))
+
+### Refactoring
+* Created a new internal data model, Events, to capture data as it flows through the pipeline. This was introduced to eliminate the excessive de/serialization of the current implementation. Currently, integrated with only log ingestion and sample plugins ([#](https://github.com/opensearch-project/data-prepper/pull/412)), ([#435](https://github.com/opensearch-project/data-prepper/pull/435)), ([#463](https://github.com/opensearch-project/data-prepper/pull/463)), ([#468](https://github.com/opensearch-project/data-prepper/pull/468)), ([#477](https://github.com/opensearch-project/data-prepper/pull/477)), & ([#539](https://github.com/opensearch-project/data-prepper/pull/539))
+* The StdOutSink supports Objects instead of Strings as part of the migration to support the new event model ([#599](https://github.com/opensearch-project/data-prepper/pull/599))
+* The FileSource supports the new event model ([#](https://github.com/opensearch-project/data-prepper/pull/601))
```

Comment:
Nit: FileSource supports Objects similarly to StdOutSink

Reply:
From a CX perspective it supports the new model.